machine/dni/dni_3048up/onie-rom.conf

```diff
@@ -0,0 +1,19 @@
+# DNI 3048UP onie ROM configuration
+
+description="Delta Networks, DNI-3048UP"
+
+# Top Down the Nand flash looks like:
+# 1. u-boot image  (include shmoo)
+# 2. u-boot environment
+# 3. onie-uimage
+format=contiguous-up
```

Comment:
Is the format really contiguous-up.
From dts entries , board_eeprom seems to exist between uboot-env and ONIE
Format should be uboot_ubootenv-up and please refer to qemu_armv7a for reference 
which has similar layout
 
https://github.com/opencomputeproject/onie/blob/master/machine/qemu_armv7a/INSTALL
If this